ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -22,10 +22,6 @@
 //!  2. or, when they need extra information, they expose a `display()` method, which returns an
 //!     object which implements [std::fmt::Display].
 
-// For now, don't swamp others working in other areas of the system.
-// FIXME: eventually delete.
-#![allow(dead_code)]
```

Comment:
OK, now I see why you are being to aggressive with removing dead, but potentially needed later code :)

Reply:
I'm transparent :)